charts/edge-dcb/values.yaml

```diff
@@ -49,7 +49,7 @@ ingress:
   hosts:
     - host: chart-example.local
       paths:
-        - path: /*
+        - path: /dcbService*
```

Comment:
updating chart version isn't required in this case? How indexing of chart will work with this change?

Reply:
Agree. My bad. Have changed.

Reply:
If I'm not mistaken common charts should also be updated

Reply:
There is no need to change the folio-common chart version as no changes have been made. The approach, "the more we hand over, the better it is" is possible but not required.